### ISSUE TITLE
La7 TV

### ISSUE THREAD

**Issue by zungunz** [yt-dlp/yt-dlp#5876]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Italy

### Provide a description that is worded well enough to be understood

Hi all.
I have seen this behavior sinche two almost months ago. Before it worked flawlessy here.
By now also oldest url's give the same broken results. I mean oldest that two months url's.
Thus it seems that the site of la7, a free to air private italian television, had become the problem source for the whole batch of videos that store.

  Thanks in advance for any support.

                           Zun

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
use@host1:~
$ yt-dlp -vU https://www.la7.it/propagandalive/rivedila7/propaganda-live-best-17-12-2022-464815
[debug] Command-line config: ['-vU', 'https://www.la7.it/propagandalive/rivedila7/propaganda-live-best-17-12-2022-464815']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (zip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.17.0-3mx-amd64-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: none
[debug] Optional libraries: certifi-2020.06.20, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [la7.it] Extracting URL: https://www.la7.it/propagandalive/rivedila7/propaganda-live-best-17-12-2022-464815
[la7.it] propaganda-live-best-17-12-2022-464815: Downloading webpage
[la7.it] propaganda-live-best-17-12-2022-464815: Downloading MPD manifest
WARNING: [la7.it] Failed to download MPD manifest: HTTP Error 413: Request Entity Too Large
[la7.it] propaganda-live-best-17-12-2022-464815: Downloading m3u8 information
WARNING: [la7.it] Failed to download m3u8 information: HTTP Error 413: Request Entity Too Large
ERROR: [la7.it] propaganda-live-best-17-12-2022-464815: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1485, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1582, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1641, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 2573, in process_video_result
    self.raise_no_formats(info_dict)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1035, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [la7.it] propaganda-live-best-17-12-2022-464815: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

use@host1:~
```

**Comment by bashonly** [comment]
This issue has been fixed in master (72f96c55662c688a15ed00ffa661546156f7e461). If you don't want to wait for the next release, you can [install the master branch](https://github.com/yt-dlp/yt-dlp/wiki/Installation#with-pip) or try the [unofficial daily builds](https://github.com/ytdl-patched/yt-dlp).

Closing as duplicate of #5360

**Comment by zungunz** [comment]
TU and season greetings!!!

        Zun
